enterprise/server/self-hosting-hunspell.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: default
+title: Add Hunspell dictionaries to Spell Checker Pro
+title_nav: Using Hunspell dictionaries
+description: Instructions for adding Hunspell dictionaries to TinyMCE Spell Checker Pro.
+keywords: enterprise tinymcespellchecker spell check checker pro server configuration configure
```

Comment:
I assume the search doesn't need Hunspell in keywords because it's already in the title/description?

Reply:
Pretty sure these keywords aren't used anywhere anymore (they use to be in the head
```suggestion
keywords: enterprise tinymcespellchecker hunspell spell check checker pro server configuration configure
```